crates/interpreter/src/module.rs

```diff
@@ -27,12 +28,14 @@ use crate::*;
 pub struct Module<'m> {
     binary: &'m [u8],
     types: Vec<FuncType<'m>>,
+    // TODO(dev/fast-interp): Change to `&'m [SideTableEntry]` when making it persistent in flash.
+    side_tables: Vec<SideTableEntry>,
```

Comment:
There's at least 2 options here: `[SideTableEntry]` or `[[SideTableEntry]]`. The first one assumes that each function would have an index to its first entry (and the side tables of all functions are concatenated). That's probably the simplest once we persist to flash. The second option is one side table `[SideTableEntry]` per function, hence the second layer. The side table of a function would be at the same outer index as the function index.

Reply:
Good catch. I intended to make it a 2D vector as `side_tables` was plural, but forgot to do so.